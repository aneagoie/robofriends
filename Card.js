import React from 'react';
import CardList from './CardList';

const Card = (props) => {
	const {name, email, id} = props;
  return(
	<div className='bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5'>
	  <img alt='robots' src={`https://robohash.org/${props.id}?200x200`} />
	  <div>
		<h2>{name}</h2>
		<p>{email}</p>
	 </div>
	</div>
  );
}

export default Card;