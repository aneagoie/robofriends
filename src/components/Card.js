import React from 'react';
const Card =({AboutRobo,name,email,id,key1})=>{
	return(
		<button onClick={(ke)=>AboutRobo(ke,key1)} className = "ma4 col-1-of-4 card mrg-top500 bg-light-blue card__design  grow" >
				<img className = "card card__photo" src = {`https://robohash.org/${id}`} alt = 'robo img'/>
				<div className = "card card__details u-center-text">
					<h2>{name}</h2>
					<p className="card__details--email">{email}</p>
				</div>
		</button>          
	);
}
export default Card