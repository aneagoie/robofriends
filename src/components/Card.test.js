import React from 'react'

import { shallow } from "enzyme";
import Card from "./Card";

it('expect to render Card component', () => {
  expect(shallow(<Card/>).length).toEqual(1)
})
