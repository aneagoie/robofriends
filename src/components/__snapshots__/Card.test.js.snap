// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
